### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'toString')

### ISSUE THREAD

**Issue by yjose** [jamesacarr/eslint-formatter-github-actions#18]
Hey, thank you @jamesacarr  for the amazing work on the formatter. 
i am using if for while now and seem to work perfectly, but today start throw an error  `TypeError: Cannot read properties of undefined (reading 'toString')`

Any recommendation on how to fix this issue  

![image](https://user-images.githubusercontent.com/11137944/220372099-81812f7c-abd4-4c28-aa66-92c18dfa2ef6.png)

**Comment by jamesacarr** [comment]
@yjose this package is pretty out of date because I haven't touched it in a while. I'll see if I can get it updated for you over the next day or two, which will hopefully fix the error you're seeing.

**Comment by acidjazz** [comment]
same error

```
Oops! Something went wrong! :(
  
  ESLint: 8.36.0
  
  TypeError: Cannot read properties of undefined (reading 'toString')
      at /home/runner/work/web/web/node_modules/@jamesacarr/eslint-formatter-github-actions/dist/index.js:17:36
      at Array.map (<anonymous>)
      at /home/runner/work/web/web/node_modules/@jamesacarr/eslint-formatter-github-actions/dist/index.js:[12](https://github.com/vulncheck/web/actions/runs/4445738357/jobs/7805214716#step:6:13):68
      at Array.flatMap (<anonymous>)
      at format (/home/runner/work/web/web/node_modules/@jamesacarr/eslint-formatter-github-actions/dist/index.js:12:32)
      at Object.format (/home/runner/work/web/web/node_modules/eslint/lib/eslint/eslint.js:637:24)
      at printResults (/home/runner/work/web/web/node_modules/eslint/lib/cli.js:252:36)
      at async Object.execute (/home/runner/work/web/web/node_modules/eslint/lib/cli.js:443:[13](https://github.com/vulncheck/web/actions/runs/4445738357/jobs/7805214716#step:6:14))
      at async main (/home/runner/work/web/web/node_modules/eslint/bin/eslint.js:135:[24](https://github.com/vulncheck/web/actions/runs/4445738357/jobs/7805214716#step:6:25))
```

**Comment by acidjazz** [comment]
@jamesacarr i see you commited a fix - mind snapping/tagging us a v0.1.1 ?

**Comment by jamesacarr** [comment]
Hey @acidjazz (and @yjose), just released the fix to NPM as 0.2.0. Sorry about the delay there, completely forgot to actually publish it afterwards.

Let me know if you still run into any issues.
